Microsoft.DotNet.Interactive.Tests/LanguageKernelTests.cs

```diff
@@ -886,5 +887,35 @@ public async Task PowerShell_streams_handled_in_correct_order()
                     .Value.ToString().Should().Be(errorMessage),
                 e => e.Should().BeOfType<CommandHandled>());
         }
+
+        [Fact(Timeout = 45000)]
```

Comment:
You don't need to add the timeouts.

Reply:
All the other tests in this file have timeouts for similar tests. Should remove those as well?

Reply:
If you want to, feel free. If not, we'll do it. These were an artifact of an investigation into timeout and memory issues that were plaguing us but have been resolved.

Reply:
I removed them just from my added tests.